src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -24,20 +26,22 @@ trait WorkspaceService extends HttpService with PerRequestCreator with FireCloud
   lazy val log = LoggerFactory.getLogger(getClass)
   lazy val rawlsWorkspacesRoot = FireCloudConfig.Rawls.workspacesUrl
 
+  def transformSingleWorkspaceRequest(entity: HttpEntity): HttpEntity = {
+    entity.as[RawlsWorkspaceResponse] match {
+      case Right(rwr) => new UIWorkspaceResponse(rwr).toJson.prettyPrint
```

Comment:
Ah, you're right, just `new UIWorkspaceResponse(rwr).toJson.toString` should work.